server_proxy/src/pxtcp.c

```diff
@@ -3951,7 +3952,7 @@ int ism_transport_startTCPEndpoint(ism_endpoint_t * endpoint) {
 int ism_transport_startTCP(void) {
     xUNUSED int    rc;
     int    i;
-    char   threadname[16];
+    char   threadname[18];
```

Comment:
Where does 18 come from? I'm guessing it is essentially "tcpiop.2147483647" plus null terminator? (ie if we hit maxint number of processes) Can you add a comment explaining it.

Reply:
Yes that is correct.
To avoid truncation warnings by setting buffer to 18 bytes. "tcpiop.(number)" tcpiop takes 7 characters, number which is the variable part can take largest number upto 10 digit long and 1 character for the null terminator.

Reply:
Agree s comment would be an improvement

Reply:
@jonquark should I add the comment in the code itself explaining the change?

Reply:
Yes something like:
    char   threadname[18]; //Can be up to tcpiop.2147483647+null in length

Reply:
Done, added the comment.